stacks/gopaddle-lite/upgrade.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add gopaddle https://gopaddle-io.github.io/gopaddle-lite/
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="gopaddle-lite"
+CHART="gopaddle/gopaddle-lite"
```

Comment:
Looks like the stack and chart here are wrong - running as is gives:
"Error: chart "gopaddle-lite" matching  not found in gopaddle index. (try 'helm repo update'): no chart name found"
Updating them to be 
```
STACK="gp-lite-4-2"
CHART="gopaddle/gp-lite"
```
lets the upgrade script succeed.